tests/state_test.go

```diff
@@ -706,10 +706,20 @@ func TestAllETH(t *testing.T) {
 	skipTests["RevertPrefoundEmptyOOG.json"] = "State trie clearing unimplemented"
 	skipTests["FailedCreateRevertsDeletion.json"] = "State trie clearing unimplemented"
 
+	skipTests["create2noCash.json"] = "Bug"
+	skipTests["stCreate2/Constantinople"] = "Only testing for ConstantinopleFix"
+
 	unsupportedDirs := map[string]bool{
 		"stZeroKnowledge":  true,
 		"stZeroKnowledge2": true,
-		"stCreate2":        true,
+		"stReturnDataTest": true,
```

Comment:
Why are we skipping so many directories?

Reply:
I mentioned in the other PR on eth-classic: Shouldn't be skipping stReturnDataTest, stSStoreTest, stBugs, or stCodeSizeLimit